doc/Gabc.tex

```diff
@@ -209,6 +209,29 @@ \subsection{Neume Fusion}
 (;) @<v>[</v>IJkh<v>]</v>(@[IJkh])
 }
 
+\subsection{Stem length for the bottom lines}
+
+The length of the stem is deterministically determined for most neumes.
```

Comment:
I would change this to something like `Gregorio will determine the length of the stem for most neumes.`